src/queries.js

```diff
@@ -375,6 +376,38 @@ function getByDisplayValue(...args) {
   return firstResultOrNull(getAllByDisplayValue, ...args)
 }
 
+function makeFinder(getter) {
```

Comment:
how about a comment here like

```suggestion
// wraps the get query with an awaitable
// version that retries until a timeout
function makeFinder(getter) {
```

Reply:
Excellent 👍